zulint/custom_rules.py

```diff
@@ -32,8 +32,17 @@ class RuleList:
     """Defines and runs custom linting rules for the specified language."""
 
     def __init__(self, langs, rules, max_length=None, length_exclude=set(), shebang_rules=[],
-                 exclude_files_in=None):
-        # type: (List[str], List[Rule], Optional[int], Set[str], List[Rule], Optional[str]) -> None
+                 exclude_files_in=None, exclude_max_length_fns=['test', 'example', 'migrations'],
+                 exclude_max_length_line_patterns=[
+                    '# type',
+                    '# ignorelongline',
+                    # Don't throw errors for markdown format URLs
+                    r"^\[[ A-Za-z0-9_:,&()-.]*\]: http.*",
+                    # Don't throw errors for URLs in code comments
+                    r"[#].*http.*",
+                    r"`\{\{ api_url \}\}[^`]+`",
+                ]):
```

Comment:
Can these default values by the empty string, and just moved to the `zulip/zulip` configuration?

Reply:
@timabbott done! Sorry, I lost track of this.